src/CommunityToolkit/Xamarin.CommunityToolkit/ObjectModel/Extensions/INotifyPropertyChangedExtension.shared.cs

```diff
@@ -3,7 +3,7 @@
 
 namespace Xamarin.CommunityToolkit.ObjectModel.Extensions
 {
-	public static class INotifyPropertyChangedEx
+	public static class INotifyPropertyChangedExtension
```

Comment:
Looks like that you missed this one

Reply:
This one is related to this comment: 
![image](https://user-images.githubusercontent.com/6609929/108113308-0a9be800-70a0-11eb-89f3-275cc1c0a157.png)

If one extension method should be renamed, all of them should. Is this renaming also needs to be done in a separate PR? Can it al least be the same PR with the previous reverted change?

Reply:
Yes, we can pass this one here.